scipy/io/matlab/mio5_utils.pyx

```diff
@@ -31,7 +31,6 @@ cdef extern from "Python.h":
 from cpython cimport PyBytes_Size, PyBytes_FromString
```

Comment:
```suggestion
from cpython cimport PyBytes_Size, PyUnicode_FromString
```

Reply:
I am having trouble importing this. The build fails, saying `'cpython/PyUnicode_FromString.pxd' not found`. Can you help me with how to import it correctly?

Reply:
Oh, it seems cython is missing declarations for this function. Just put on line 28:
```
cdef extern from "Python.h":
    unicode PyUnicode_FromString(const char *u)  # <-- Add this declaration
    ctypedef struct PyTypeObject:
        pass
```

Reply:
I'm sorry this is still giving the same error. The entire error thread is below:
```
Error compiling Cython file:
------------------------------------------------------------
...
cdef extern from "Python.h":
    unicode PyUnicode_FromString(const char *u)
    ctypedef struct PyTypeObject:
        pass

from cpython cimport PyBytes_Size, PyUnicode_FromString
^
------------------------------------------------------------

mio5_utils.pyx:32:0: 'cpython/PyUnicode_FromString.pxd' not found
Traceback (most recent call last):
  File "/home/namami2011/dev/scipy/tools/cythonize.py", line 324, in <module>
    main()
  File "/home/namami2011/dev/scipy/tools/cythonize.py", line 320, in main
    find_process_files(root_dir)
  File "/home/namami2011/dev/scipy/tools/cythonize.py", line 309, in find_process_files
    for result in pool.imap_unordered(lambda args: process(*args), jobs):
  File "/home/namami2011/anaconda3/envs/scipydev/lib/python3.8/multiprocessing/pool.py", line 868, in next
    raise value
  File "/home/namami2011/anaconda3/envs/scipydev/lib/python3.8/multiprocessing/pool.py", line 125, in worker
    result = (True, func(*args, **kwds))
  File "/home/namami2011/dev/scipy/tools/cythonize.py", line 309, in <lambda>
    for result in pool.imap_unordered(lambda args: process(*args), jobs):
  File "/home/namami2011/dev/scipy/tools/cythonize.py", line 243, in process
    processor_function(fromfile, tofile, cwd=path)
  File "/home/namami2011/dev/scipy/tools/cythonize.py", line 108, in process_pyx
    raise Exception('Cython failed')
Exception: Cython failed
Traceback (most recent call last):
  File "setup.py", line 619, in <module>
    setup_package()
  File "setup.py", line 603, in setup_package
    generate_cython()
  File "setup.py", line 354, in generate_cython
    raise RuntimeError("Running cythonize failed!")
RuntimeError: Running cythonize failed!
```

Can this be fixed?

Reply:
The function declaration should replace the import.

Reply:
Ok, got it! I am still learning cpython 😅. It's now working. Thank you!